.github/workflows/install.yml

```diff
@@ -0,0 +1,39 @@
+name: Install
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+    tests:
+      name: Test installation
+      runs-on: ubuntu-latest
+
+      steps:
+        - name: Checkout code
+          uses: actions/checkout@v2
+          with:
+            submodules: recursive
+        - name: Set system to non-interactive mode
+          run: export DEBIAN_FRONTEND=noninteractive
+        - name: install dependencies
+          run: |
+            sudo apt-get install -y --force-yes -qq build-essential
+        - name: build and install
+          run: |
+            mkdir -p build
+            cd build
+            mkdir -p ${HOME}/install
+            echo "Build"
+            cmake -DCMAKE_INSTALL_PREFIX=${HOME}/install ..
```

Comment:
Remove `mkdir` and `cd` above and just use `cmake -B build` here and `cmake --build build` instead of `make`

Reply:
Good point. Done.